ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -168,6 +168,31 @@ public struct AvatarView: View {
         self.tokens = state.tokens
     }
 
+    public init(style: MSFAvatarStyle,
```

Comment:
I agree with @DrTacoMD that we should not add an initializer parameter of type MSAvatarState and copy the properties.

I'm leaning towards pursuing the possibility of storing the AvatarView types in the state object and just expose their states publicly.
Then the rendering body property will just use the AvatarViews inside the state object here.

Can you explore this option?

Reply:
Interesting, but I think I'd push back a little bit on this. There's a conceptual clarity that comes from our current separation, where state objects don't contain SwiftUI views directly, so all ownership is unidirectional. If we store `AvatarView` objects inside the state then we invert that relationship.